core/templates/pages/topic-editor-page/navbar/topic-editor-navbar.directive.html

```diff
@@ -54,43 +54,79 @@
 </ul>
 <div class="topic-nav-mobile">
   <div class="topic-navbar-mobile-tabs">
-    <div class="save-change-btn-container">
-      <button class="btn save-changes-btn"
-              ng-click="saveChanges()">
-        <span ng-if="isSaveInProgress()">
-          <span>Saving</span>
-        </span>
-        <span ng-if="!isSaveInProgress()">
-          Save Changes
-        </span>
-        <span ng-if="getChangeListLength() > 0">
-              (<[getChangeListLength()]>)
-        </span>
-      </button>
-      <div class="arrow-up-container" ng-click="toggleDiscardChangeButton()">
-        <i class="fa fa-arrow-up"></i>
+    <div class="dropdown-container">
```

Comment:
"dropdown-container", "dropdown-options", "dropdown-text" etc. seem like generic class names. If these classes are not going to be used site-wide, can we change the name to scope it to this directive?

Reply:
Yeah right. Done.